Squeezer/src/main/java/uk/org/ngo/squeezer/service/HomeMenuHandling.java

```diff
@@ -180,76 +172,36 @@ void addNode(JiveItem jiveItem, List<JiveItem> homeMenu) {
         }
     }
 
+    private static int CUSTOM_SHORTCUT_WEIGHT = 2000;
+    private static String CUSTOM_SHORTCUT_NODE = "home";
+
     /**
      * Load complete list of stored items from preferences.
      * Use action the values on the initialized customNodes.
      */
     public void loadShortcutItems() {
```

Comment:
HomeMenuHandling needs "new Preferences(Squeezer.getContext())", this indicates an issue.
E.g. setHomeMenu get the archived items from the caller but loadShortcutItems reads shortcut items itself.
Either HomeMenuHandling should only handle the home menu, or it should do all the load and save of archive and shortcuts.

Reply:
Done. However, I am not satisfied with e.g. SqueezeService line 1302.